plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -142,6 +134,37 @@ class Bazel(
         )
     }
 
+    /**
+     * This function determines the Bazel module registry to use based on the given [lockfile]: If this is a lockfile
+     * generated by Bazel version >= 7.2.0, a [CompositeBazelModuleRegistryService] based on the "registryFileHashes"
+     * will be returned. Else, either a [LocalBazelModuleRegistryService] or a [RemoteBazelModuleRegistryService] based
+     * on the "cmdRegistries" will be returned.
+     */
+    private fun determineRegistry(lockfile: Lockfile, projectDir: File): BazelModuleRegistryService {
+        // Bazel version < 7.2.0.
+        if (lockfile.flags != null) {
+            val registryUrl = lockfile.registryUrl()
+
+            return registryUrl.withoutPrefix("file://")?.let {
+                val localRegistryURL = it.replace("%workspace%", projectDir.absolutePath)
```

Comment:
Note to myself: This code seems to be redundant with the one in the composite registry.

Reply:
Yes it can be refactored after https://github.com/oss-review-toolkit/ort/issues/8801, since I suspect it will change the code a bit.